internal/sqlchannel/client.go

```diff
@@ -138,6 +138,41 @@ func (cli *OperationClient) GetSystemAccounts() ([]string, error) {
 	return result, err
 }
 
+func (cli *OperationClient) JoinMember() error {
```

Comment:
should provide interface definitions for all these operations, or the mock will be difficult to implement

Reply:
the caller should be able to pass in a timeout